static/app/views/starfish/components/detailPanel.tsx

```diff
@@ -39,6 +41,12 @@ export default function Detail({
   );
   const escapeKeyPressed = useKeyPress('Escape');
 
+  useEffect(() => {
```

Comment:
Can you just initialize the state in the component to whatever the default is? If the component is open by default, useEffect will cause it to jump from right to bottom side of the screen because useEffect is async